moduleVNext/Entra/config/ModuleMetadata.json

```diff
@@ -0,0 +1,35 @@
+{
+    "guid": "742dccd1-bf4b-46a0-a3f2-14e0bb508233",
+    "authors": "Microsoft",
+    "owners": "Microsoft",
+    "description": "Microsoft Entra PowerShell v1.0: Microsoft Entra PowerShell is a scenario-driven module that allows administrators to efficiently manage Entra ID resources, including users, groups, applications, and policies, among others",
```

Comment:
Pleased to see it referred to as 'Microsoft Entra PowerShell.' Would be ideal for existing strings in the module's source and documentation to be changed from 'Microsoft.Graph.Entra' to 'Microsoft.Entra' and 'Microsoft.Graph.Entra.Beta' to 'Microsoft.Entra.Beta' (but I understand that change will take some time to implement).